pom.xml

```diff
@@ -562,15 +560,41 @@
                     </executions>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-remote-resources-plugin</artifactId>
```

Comment:
Interesting. I didn't know this plugin and it's in fact very useful... Nice trick.